### ISSUE TITLE
derive macro warning: unexpected `cfg` condition name: `wasm_bindgen_unstable_test_coverage`

### ISSUE THREAD

**Issue by mirsella** [rustwasm/wasm-bindgen#4283]
version: `wasm-bindgen v0.2.95`
Recent rust nightly version enabled by default the `unexpected_cfgs`, which lead to this warning:
```
warning: unexpected `cfg` condition name: `wasm_bindgen_unstable_test_coverage`
  --> game/src/lib.rs:29:1
   |
29 | #[wasm_bindgen]
   | ^^^^^^^^^^^^^^^
   |
   = help: consider using a Cargo feature instead
   = help: or consider adding in `Cargo.toml` the `check-cfg` lint config for the lint:
            [lints.rust]
            unexpected_cfgs = { level = "warn", check-cfg = ['cfg(wasm_bindgen_unstable_test_coverage)'] }
   = help: or consider adding `println!("cargo::rustc-check-cfg=cfg(wasm_bindgen_unstable_test_coverage)");` to the top of the `build.rs`
   = note: see <https://doc.rust-lang.org/nightly/rustc/check-cfg/cargo-specifics.html> for more information about checking conditional configuration
   = note: this warning originates in the attribute macro `wasm_bindgen` (in Nightly builds, run with -Z macro-backtrace for more info)
```

**Comment by bcheidemann** [comment]
For anyone else coming across this, you can [statically declare the cfg as expected](https://blog.rust-lang.org/2024/05/06/check-cfg.html#expecting-custom-cfgs) by adding the following to your `Cargo.toml`:

```toml
[lints.rust]
unexpected_cfgs = { level = "warn", check-cfg = ['cfg(wasm_bindgen_unstable_test_coverage)'] }
```

**Comment by daxpedda** [comment]
Fixed by #4284.

**Comment by mirsella** [comment]
nice, that's a better fix than just ignoring the lint

**Comment by mirsella** [comment]
since this can break CI by emitting a warning to potentially a lot of people, it would be nice to have a patch release
thanks and have a good day :)

**Comment by daxpedda** [comment]
> since this can break CI by emitting a warning to potentially a lot of people, it would be nice to have a patch release thanks and have a good day :)

I plan to make one today or at the latest tomorrow.

**Comment by MingweiSamuel** [comment]
This doesn't seem to be fixed? 0.2.100

`warning: unexpected `cfg` condition name: `wasm_bindgen_unstable_test_coverage``
